src/connectdlg.cpp

```diff
@@ -41,51 +41,63 @@ CConnectDlg::CConnectDlg ( CClientSettings* pNSetP, const bool bNewShowCompleteR
     setupUi ( this );
 
     // Add help text to controls -----------------------------------------------
-    // server list
+    // directory
+    QString strDirectoryWT = "<b>" + tr ( "Directory" ) + ":</b> " +
+                             tr ( "Selects the server list to be shown. "
```

Comment:
Active voice is preferred, so "Selects the list of servers" perhaps?

Reply:
"Shows the servers listed by the directory"?  (I'm trying to get that idea into people's heads 😁 )

Reply:
"...selected directory" even.